### ISSUE TITLE
Identification of the selected element

### ISSUE THREAD

**Issue by dipisoft** [faush01/playback_reporting#13]
Hi,

In the "Played" tab, the list of films/folders does not identify the selected item. Which would be handy though.

To work around this small problem I added this in "Custom CSS" of the "Settings" page of Emby:

`#item_list tr:hover { background-color: green; color: white; }`

On the other hand, it would be handy to be able to visually distinguish whether an item is a movie or a folder.

Thank you in advance and congratulations for this very practical extension!

**Comment by faush01** [comment]
I had forgotten I had even added that tab, it was a Sunday afternoon project I worked on and it kind of worked well enough so I added it, it could probably do with some extra work but to be honest your above feedback is the first time anyone at all ever has commented on this tab so as I said I had forgotten about it.

I will try to set aside some time to play with this, there is another plugin project (ChapterAPI) I worked on that uses a similar control to show lists of movies tv shows etc that I think has a better implementation that I might be able to adapt. From memory this selector was the bases of that later work so it it is only fair I feed some of that work back into this feature.
